smdebug/tensorflow/tensor_ref.py

```diff
@@ -19,10 +19,12 @@ def get_tf_names(arg):
         tf_names = [arg.name]
     elif isinstance(arg, tf.Tensor):
         tf_names = [arg.name]
-    elif isinstance(arg, values.DistributedVariable):
+    elif isinstance(arg, values.DistributedValues):
```

Comment:
make sure you replace all occurrences of DistributedVariable. also, make sure it doesn't break the training script reported on a recent internal ticket

Reply:
Done. Thanks for this